MapboxNavigation/NavigationMapView.swift

```diff
@@ -1011,6 +1011,22 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
             style.addLayer(symbol)
         }
     }
+    
+    @objc public func enableOverheadCameraView(from userLocation: CLLocationCoordinate2D, along coordinates: [CLLocationCoordinate2D], for bounds: UIEdgeInsets) {
```

Comment:
`enable` indicates that the behavior is persistent but it's just a one-time update of the coordinate bounds, right? How about `update` or `set`?